infra/recipes/terraform/azure/environments/sandbox/README.md

```diff
@@ -53,6 +67,33 @@ _If using environment variables_
 
 `terraform plan` to see the plan generated by terraform for the specified deployment configuration. If you are satisfied with the plan, run `terraform apply` to apply the configuration.
 
+You will need to run `terraform plan` and `terraform apply` from two dirrent directories. First, run the commands from the `../container/` directory to set up the Azure Container Registry. Then, run the commands from the `sandbox/` (current) directory to set up the rest of the sandbox.
+
 ## Deconfiguration
 
-Run `terraform destroy` once you are done to tear down the services you have created.
\ No newline at end of file
+Run `terraform destroy` once you are done to tear down the services you have created.
+
+# Access Created Services
+
+Once you have successfully run the above steps so the sandbox has been run, you can now access the pods in AKS in your local terminal by ensuring you are logged in to the Azure CLI with `az login` and then running
+`az aks get-credentials --resource-group <resource-group> --name <cluster-name>`. You can then run any `kubectl` commands.
+
+## Make requests to services
+
+To make requests to the services, set up port forwarding by running `kubectl port-forward <pod-name> 8080:<destination-port>`. You can get the pod name by running `kubectl get pods`. Then, you can make requests to HTS and Tables service via `http://localhost:<destination-port>`. If you want to make requests to both HTS and Tables services you will need to forward to two separate ports.
+
+## Run client shell
+
+### Start local spark cluster
+TODO: MAKE BETTER, UNIQUE FOR SPARK CLUSTER
```

Comment:
Is it not possible to run the spark cluster on Azure?

Reply:
We were running into issues with the HDInsight cluster and the cores required, so we decided to do local instead